subprojects/packagefiles/speexdsp/libspeexdsp/meson.build

```diff
@@ -0,0 +1,53 @@
+sources = [
+  'buffer.c',
+  'fftwrap.c',
+  'filterbank.c',
+  'jitter.c',
+  'kiss_fft.c',
+  'kiss_fftr.c',
+  'mdf.c',
+  'preprocess.c',
+  'resample.c',
+  'scal.c',
+  'smallft.c',
+]
+
+cc_is_not_msvc = cc.get_id() != 'msvc'
+
+m_dep = cc.find_library('m', required : cc_is_not_msvc)
+
+speexdsp = library('speexdsp',
+  sources,
+  dependencies: m_dep,
+  include_directories : incdir,
+  vs_module_defs: '../win32/libspeexdsp.def',
+  version: lib_version,
+  soversion: cc_is_not_msvc ? lib_soversion : '',
```

Comment:
Why is this msvc check needed?

Reply:
It came from the `ogg` wrap:

``` text
subprojects/packagefiles/ogg/src/meson.build:  soversion: cc.get_id() == 'msvc' ? '' : lib_soversion,
```

(but now that I've stripped the auto-tools `lib_`* baggage, this is all going away)